dipy/tracking/local/tissue_classifier.pyx

```diff
@@ -19,7 +61,7 @@ cdef class ThresholdTissueClassifier(TissueClassifier):
 
     def __cinit__(self, metric_map, threshold):
         self.interp_out_view = self.interp_out_double
-        self.metric_map = metric_map
+        self.metric_map = metric_map.astype('float64')
```

Comment:
I think it's fine if you meant to do this, but `astype` always makes a copy of the array. Are you trying to make a copy here or would a view be acceptable when the array is already `float64`? `np.asarray(metric_map, 'float64')` will enforce the type, but return a view if possible.